doc/xsite.md

```diff
@@ -38,7 +38,7 @@ spec:
         url: openshift://api.site-a.devcluster.openshift.com:6443
```

Comment:
I think we should remove this file

Reply:
Yeah, I was of two minds about removing the file or not at first. It's probably cleaner to remove it so we don't run the risk of out-of-date docs or community users following the wrong examples.